### ISSUE TITLE
[Mandrel 22.3.1] CXF SOAP native image compilation failure

### ISSUE THREAD

**Issue by jamesnetherton** [apache/camel-quarkus#4467]
```
2023-02-01T04:15:36.7285943Z Error: Class initialization of org.apache.neethi.builders.converters.OMToDOMConverter failed. This error is reported at image build time because class org.apache.neethi.builders.converters.OMToDOMConverter is registered for linking at image build time by command line Use the option --initialize-at-run-time=org.apache.neethi.builders.converters.OMToDOMConverter to explicitly request delayed initialization of this class.
2023-02-01T04:15:36.7286881Z Detailed message:
2023-02-01T04:15:36.7287018Z 
2023-02-01T04:15:36.7288222Z com.oracle.svm.core.util.UserError$UserException: Class initialization of org.apache.neethi.builders.converters.OMToDOMConverter failed. This error is reported at image build time because class org.apache.neethi.builders.converters.OMToDOMConverter is registered for linking at image build time by command line Use the option --initialize-at-run-time=org.apache.neethi.builders.converters.OMToDOMConverter to explicitly request delayed initialization of this class.
2023-02-01T04:15:36.7289111Z Detailed message:
2023-02-01T04:15:36.7289222Z 
2023-02-01T04:15:36.7289466Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.UserError.abort(UserError.java:85)
2023-02-01T04:15:36.7290021Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.FallbackFeature.reportAsFallback(FallbackFeature.java:248)
2023-02-01T04:15:36.7290656Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:776)
2023-02-01T04:15:36.7291277Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:578)
2023-02-01T04:15:36.7291838Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:535)
2023-02-01T04:15:36.7292449Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:403)
2023-02-01T04:15:36.7293097Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:580)
2023-02-01T04:15:36.7293727Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:128)
2023-02-01T04:15:36.7295243Z Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Class initialization of org.apache.neethi.builders.converters.OMToDOMConverter failed. This error is reported at image build time because class org.apache.neethi.builders.converters.OMToDOMConverter is registered for linking at image build time by command line Use the option --initialize-at-run-time=org.apache.neethi.builders.converters.OMToDOMConverter to explicitly request delayed initialization of this class.
2023-02-01T04:15:36.7296217Z Detailed message:
2023-02-01T04:15:36.7296331Z 
2023-02-01T04:15:36.7296666Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.constraints.UnsupportedFeatures.report(UnsupportedFeatures.java:126)
2023-02-01T04:15:36.7299669Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:771)
2023-02-01T04:15:36.7300087Z 	... 5 more
2023-02-01T04:15:36.7300544Z Caused by: java.lang.NoClassDefFoundError: org/apache/axiom/om/OMException
2023-02-01T04:15:36.7300970Z 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
2023-02-01T04:15:36.7301420Z 	at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1155)
```

**Comment by jamesnetherton** [comment]
FYI @ppalaga.

**Comment by zhfeng** [comment]
@jamesnetherton @ppalaga any update?

**Comment by ppalaga** [comment]
Sorry for the delay, let me have a look.

**Comment by ppalaga** [comment]
I have run all native tests under `integration-test-groups/cxf-soap` and also `integration-tests/cxf-soap-grouped` and I do not see any issue. 
@jamesnetherton do you happen to remember if there are some special steps to reproduce?

**Comment by ppalaga** [comment]
OK, I see 975cb3b270692778854d8eb6134582850020fba2 is in quarkus-main. Let me try that one.

**Comment by ppalaga** [comment]
> OK, I see [975cb3b](https://github.com/apache/camel-quarkus/commit/975cb3b270692778854d8eb6134582850020fba2) is in quarkus-main. Let me try that one.

I see a different error there which goes away when I upgrade to QCXF 2.3.0-SNAPSHOT which I am going to release right now.

**Comment by ppalaga** [comment]
In https://github.com/apache/camel-quarkus/pull/5190, the CI fails due to other issues, but CXF seems to pass. So I'd consider this issue as solved by the upgrade to Quarkus CXF 2.3.0
